app/models/issue_tta_report.rb

```diff
@@ -15,8 +15,8 @@ class IssueTtaReport < ApplicationRecord
 
   def api_call_succeeded
     # do not make the api call here, if @activity_data is nil then we didn't attempt to make a call
-    if !@activity_data.nil? && !@activity_data[:success]
-      errors.add(:base, tta_link_error_message(@activity_data[:code]))
+    if !@activity_data.nil? && @activity_data.failed?
```

Comment:
[question] in what cases would @activity_data be nil? it looks like if it doesn't exist than we hit the endpoint.

Reply:
Any cases where we called `.valid?` or `.save` without updating the display id. I don't believe that's an actual code-path currently.